docs/quickstart.rst

```diff
@@ -236,7 +236,7 @@ Congratulations, you can now run migrations with *Septentrion*!
 Going further
 -------------
 
-To continue with practical steps, head to the :ref:`How-to... <howto>` section.
+To continue with practical steps, head to the :ref:`How-to... <how-to>` section.
```

Comment:
mmmh are you sure, what is the target of this ref now ?
Should we have a index.rst in the howto folder ?

Reply:
The target in still the « How-to... » section. Because it is referenced as `.. _how-to:` in the `howto_index.rst`, we need to use `how-to` here (not `howto`). Here is a picture of what you get when you click on the link:

![image](https://user-images.githubusercontent.com/16255570/93738508-3fd20600-fbe6-11ea-8d1c-2e6b4faf3928.png)